Front/src/app/services/products.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ProductsModel } from '../models/ProductsModel';
+import { HttpResponseModel } from '../models/HttpResponseModel';
+import { AppConfig } from '../common/app-config';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductsService {
+
+  constructor(private htpp: HttpClient) { }
+
+  ServiceEndpoint: string = `${AppConfig.API_URL}`;
+
+  // Metodo para obtener todos los productos
+  GetProducts(): Observable<HttpResponseModel>{
+    return this.htpp.get<HttpResponseModel>(this.ServiceEndpoint+'/api/Products/GetProducts');
```

Comment:
Avoid repeating the controller name every time. Also, fix for the standard endpoint names.
```suggestion
    return this.htpp.get<HttpResponseModel>(this.ServiceEndpoint+'/GetAll');
```

Reply:
Se corrigió los nombre de los controladores de acuerdo a las sugerencias y se corrigió el endpoint